Bottom: 31cc00427c8ab21de284c22bf5e12d8a1a6ae4e6
Top:    471b5927512aa0497d1ce4579a2f8cb5d392ce91
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 20:03:25 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
